lib/middleware/sidekiq/only_run_while_up.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Middleware
+  module Sidekiq
+    require 'down_time_checker'
+    class OnlyRunWhileUp
+      def call(worker, job, _queue)
+        if worker.respond_to?(:downtime_checks)
+          array = worker.downtime_checks.map { |service| DownTimeChecker.new(service).down? }
+          if array.all? { |element| element == false }
```

Comment:
Can you use `.none?` here instead of this any check.

Reply:
✅

Reply:
I know this PR is closed, but wanted to associate my comment.

Is this what you were thinking? `@downtime_checks_array.none? { |element| !element }`

That's what I went with in the new PR. You'll probably see it soon!